### ISSUE TITLE
[Feature Request]: TFM Builds need to pull in the equivalent Standard toolkit built TFM

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Extended-Toolkit#72]
Next steps after #70 is in all Branches
i.e. if a "Lite" build of the extended toolkit is created, then it needs to be NuGet Linked to the `Lite` version of the Standard toolkit

**Comment by Wagnerp** [comment]
I'm sure I've done this on the `version-next` branch to keep the csproj files 'clean'

**Comment by Smurf-IV** [comment]
*Probably* something like this:
```
<!--Krypton Toolkit Import-->
    <Choose>
        <When Condition="'$(TFMs)' == 'all'">
            <ItemGroup >
                <PackageReference Include="Krypton.Toolkit.Canary" Version="6.0.2103.6" />
            </ItemGroup >
        </When>
        <Otherwise>
            <ItemGroup >
                <PackageReference Include="Krypton.Toolkit.Canary.Lite" Version="6.0.2103.6"/>
            </ItemGroup >
        </Otherwise>
    </Choose>
```

**Comment by Wagnerp** [comment]
Will add changes to the projects.

**Comment by Wagnerp** [comment]
Closing, as I think this has been 'fixed' as per #384
